src/library_pthread.js

```diff
@@ -105,6 +105,14 @@ var LibraryPThread = {
         PThread.allocateUnusedWorker();
       }
 #endif
+#if !MINIMAL_RUNTIME
+      // MINIMAL_RUNTIME takes care of calling loadWasmModuleToAllWorkers
+      // in postamble_minimal.js
+      addOnPreRun(() => {
+        addRunDependency('loading-workers')
+        PThread.loadWasmModuleToAllWorkers(() => removeRunDependency('loading-workers'));
+      });
+#endif
```

Comment:
Is this a general change to how startup is handled that could be in a separate PR?